<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pagamento</class>
 <widget class="QDialog" name="Pagamento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Finalizar</string>
   </property>
   <property name="icon">
    <iconset resource="resoruce.qrc">
     <normaloff>:/icones/icons/money.png</normaloff>:/icones/icons/money.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Fechar</string>
   </property>
   <property name="icon">
    <iconset resource="resoruce.qrc">
     <normaloff>:/icones/icons/sign-ban.png</normaloff>:/icones/icons/sign-ban.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Informa√ßoes</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="lblModelo">
      <property name="text">
       <string>Modelo:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lblPlaca">
      <property name="text">
       <string>Placa:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblCor">
      <property name="text">
       <string>Cor:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblMarca">
      <property name="text">
       <string>Marca:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="linfoPlaca">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="linfoCor">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="linfoMarca">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="linfoModelo">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDateTimeEdit" name="dtEntrada">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>194</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dtSaida">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>194</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblEntrada">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Entrada:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSaida">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Saida:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbMensalista">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Mensalista</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>671</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Pagamento</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="lblValorPagar">
      <property name="text">
       <string>Valor a paga:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QDoubleSpinBox" name="dsValor">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lblPermanencia">
      <property name="text">
       <string>Permanencia:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="linfoPermanencia">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QComboBox" name="cbPorte">
      <item>
       <property name="text">
        <string>Medio</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pequeno</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grande</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="lblPote">
      <property name="text">
       <string>Porte:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="lblTipo">
      <property name="text">
       <string>Tipo:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QComboBox" name="cbTipo">
      <item>
       <property name="text">
        <string>Carro</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Moto</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Caminhao</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>dtEntrada</zorder>
  <zorder>dtSaida</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>lblEntrada</zorder>
  <zorder>lblSaida</zorder>
  <zorder>cbMensalista</zorder>
 </widget>
 <resources>
  <include location="resoruce.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Pagamento</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Pagamento</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
